content/influxdb/v2.0/get-started.md

```diff
@@ -331,14 +331,40 @@ _To run InfluxDB in [detached mode](https://docs.docker.com/engine/reference/run
        --name influxdb \
        -p 8086:8086 \
        --volume $PWD:/root/.influxdbv2 \
-       quay.io/influxdb/influxdb:v2.0.4
+       influxdb:2.0.4
    ```
 
+### Configure InfluxDB with Docker
+
+To mount an InfluxDB configuration file and use it from within Docker:
+
+1. follow the steps above in ["Persist data outside the InfluxDB container"](#persist-data-outside-the-influxdb-container).
+
+2. Use the command below to generate the default configuration file on the host file system:
+
+    ```console
```

Comment:
Should these be `sh` instead of `console`? I'm not sure if console is a DockerHub-specific rendering thing

Reply:
Yep, we use `sh`. Not sure how `console` got in here... Probably some copypasta...

Reply:
Ah yes, probably a DockerHub thing :)